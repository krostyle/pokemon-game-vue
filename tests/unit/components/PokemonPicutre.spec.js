import { shallowMount } from '@vue/test-utils';
import PokemonPicture from '@/components/PokemonPicture.vue';


describe('PokemonPicture', () => {
    test('must match the snapshot', () => {
        const wrapper = shallowMount(PokemonPicture, {
            props: {
                pokemonId: 1,
                showPokemonName: false
            }
        })

        expect(wrapper.html()).toMatchSnapshot();
    })

    test('must show hidden picture and pokemon 1', () => {

    })

    test('should show pokemon picture if showPokemon is true', () => {

    })
})