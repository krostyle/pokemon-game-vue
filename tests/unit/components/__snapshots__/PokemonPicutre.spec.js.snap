// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonPicture must match the snapshot 1`] = `
<div class="pokemon-container" showpokemonname="false"><img class="hidden-pokemon" src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/1.svg" alt="pokemon">
  <!--v-if-->
</div>
`;
